csrankings-a.csv

```diff
@@ -1220,7 +1221,9 @@ Ana Maria Dinis Moreira,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl
 Ana Maria Paiva,Universidade de Lisboa,http://gaips.inesc-id.pt/~apaiva/Ana_Paiva_Site_2/Home.html,dTG0kWAAAAAJ
 Ana Matran-Fernandez,University of Essex,https://www.essex.ac.uk/people/matra32701/ana-matran-fernandez,_g01zM4AAAAJ
 Ana L. Milanova,Rensselaer Polytechnic Institute,http://www.cs.rpi.edu/~milanova,PsGxHG0AAAAJ
+Ana Lucia Varbanescu,University of Amsterdam,https://www.uva.nl/en/profile/v/a/a.l.varbanescu/a.l.varbanescu.html
 Ana Moreira 0001,Universidade NOVA de Lisboa,http://nova-lincs.di.fct.unl.pt/person/12,uB4MBw4AAAAJ
+Ana Oprescu,University of Amsterdam,https://cci-research.nl/author/ana-oprescu/

```

Comment:
This faculty member has two aliases in DBLP, and both must be entered. See https://dblp.org/pid/20/1283.html.

Reply:
Also, there is no Google Scholar ID here.